tests/test_lab/test_detectors.py

```diff
@@ -262,3 +271,27 @@ def test_norm_2mode_normalized():
 def test_norm_2mode_gaussian_normalized():
     leftover = Coherent(x=[2.0, 2.0]) << Coherent(x=1.0, normalize=True)[0]
     assert np.isclose(1.0, physics.norm(leftover), atol=1e-5)
+
+
+def test_homodyne_mode_kwargs():
+
+    S1 = Sgate(modes=[0], r=1, phi=np.pi / 2)
```

Comment:
Would be nice to start adding docstrings for tests as well. At least a single line just describing what it's testing. Makes it easier both to review and when adding more tests.

Reply:
Agree!